kde-apps/libkdegames/libkdegames-4.14.3-r1.ebuild

```diff
@@ -0,0 +1,21 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit kde4-base
+
+DESCRIPTION="Base library common to many KDE games"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="debug"
+
+DEPEND="
+	media-libs/libsndfile
+	media-libs/openal
+"
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-drop-phonon.patch" )
```

Comment:
Hmm but judging by the code, it uses phonon *only* if openal&sndfile are not available, isn't that correct?